content/developer/reference/frontend/services.rst

```diff
@@ -692,6 +694,54 @@ When a rpc fails, then:
   displayed and the server is regularly contacted until it responds. The
   notification is closed as soon as the server responds.
 
+.. _frontend/services/scroller:
+
+Scroller service
+----------------
+
+Overview
+~~~~~~~~
+
+- Technical name: `scroller`
+- Dependencies: ``@web/core/utils/scrolling.js``
+
+Provides a way to handle scrolls in the webclient. 
+This service is used when a click occurs on the webclient and can
+trigger appropriate events depending of the click target.
+
+API
+~~~
+The service adds an event listener to get `click`'s on the document. The event is then
+looking to find the closest link element (``<a></a>``). If this element is coresponding to
+an anchor in the page thanks to its href attribute, the ``scrollTo`` function is used
+to handle the scroll to the correct element. That function is imported from ``@web/core/utils/scrolling.js``.
+
+.. list-table::
+    :widths: 25 25 50
+    :header-rows: 1
+
+    * - Name 
+      - Type
+      - Description
+    * - ``matchingEl``
+      - ``HTMLElement | null``
+      - The anchor targeted by the href
+
+
+.. js:function:: scrollTo(element, options)
```

Comment:
hmm, it's not a part of the scroller service, is it?

Reply:
It is not, so I don't know if and were I could document that